packages/compass-workspaces/src/types.ts

```diff
@@ -16,8 +23,14 @@ export type CollectionsWorkspace = {
 };
 
 export type CollectionWorkspace = {
+  // TODO: COMPASS-7782 use hook to get the tab id within workspace.
```

Comment:
```suggestion
  // TODO(COMPASS-7782): use hook to get the tab id within workspace.
```

Let's stick to the format we use everywhere else :)